GenerateGridJson/GenGridProgram.cs

```diff
@@ -22,7 +22,7 @@ static void Main(string[] args)
     {
         Console.WriteLine("Reading CSV and converting to JSON...");
 
-        string csvPath = "c.csv";
+        string csvPath = "Game From 2024-10-21.csv";
```

Comment:
how about using the args, so the person running can have different files.

Reply:
yes, I was focusing on initial functionallity with this commit. I will open it up to user input with a future PR. I won't do this as part of this PR.